conanfile.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from conans import ConanFile, CMake
+from conans.tools import Git
+import os
+
+class ImguiSFML(ConanFile):
+    """ Imgui-SFML library conan file.
+
+    Options:
+
+    * shared: True or False
+      Whether to build this library shared or static
+
+    * fPIC: True or False
+      Whether or not to use -fPIC option while building
+
+    * imconfig: None or String
+      Use 'None' if you want this library to provide default
+      ImGui's imconfig.h file. Otherwise, provide a path to the
+      imconfig.h file you want to include in the build.
+
+    * imconfig_install_folder: None or String
+      Use 'None' if you want this build to include imconfig.h file,
+      otherwise use anything else.
+    """
+
+    name = 'ImGui-SFML'
+    version = '2.0.1'
+    description = 'ImGui binding for use with SFML'
+    topics = ('conan', 'sfml', 'imgui')
```

Comment:
Is "Conan" a really good topic here?

Reply:
I'd also add "GUI" and maybe "game development" topic

Reply:
I took [**conan-sfml**](https://github.com/bincrafters/conan-sfml/blob/stable/2.5.1/conanfile.py) as a main example and I also checked **conan-openal** (both from bincrafters), and they do have 'conan' in topics. Though, there is no 'conan' tag at respective packages in bintray.

+1 for GUI, though.